robottelo.properties.sample

```diff
@@ -132,6 +132,9 @@ ssh_key=
 # bridge=BRIDGE_VALUE
 # gateway=GATEWAY_VALUE
 
+# For OSCAP Testing
+# [oscap]
+# content_path=~/ssg-rhel6-ds.xml
 
```

Comment:
I believe once I start to work on policies, we can add more stuff under [oscap] section.